### DISCUSSION TITLE
Ledger: "Invalid or missing bridge url parameter value"

### DISCUSSION THREAD

**Discussion by michaelbnewman** [WalletConnect/walletconnect-monorepo#1562] [Developer Support]
I'm interested in testing out WalletConnect v2 with a Ledger Nano S Plus. So far, its not working.

## Test Environment

* Ubuntu 20.04.5
* Chrome 105.0.5195.125

## Test Procedure

* Visit https://react-app.walletconnect.com/
* Testnet Only: On
* Select "Ethereum Goerli"
* Click "Connect" button.
* Click "Copy to clipboard" for QR Code. (string that starts with "wc:")
* Start Ledger Live App on desktop: ledger-live-desktop-2.49.2-linux-x86_64.AppImage
* Select Accounts
* Select Ethereum
* Click Wallet Connect button at top right
* Response: "Invalid or missing bridge url parameter value"

Also tried mainnet Ethereum via the react-app selections (Testnet Only: Off)
Same response/error.

## Other Test (React Wallet Ethers... works)

* Visit https://react-app.walletconnect.com/
* Testnet Only: On
* Select "Ethereum Goerli"
* Click "Connect" button.
* Click "Copy to clipboard" for QR Code. (string that starts with "wc:")
* React Wallet Ethers, Visit https://react-wallet.walletconnect.com/
* Click Wallet Connect icon at bottom center.
* Paste in QR code.
* Click "Connect" button.
* Choose Account 1.
* Click "Approve" button.
* Loads up fine back on react-app site.

**Comment by adrienlacombe** [comment]
hi @michaelbnewman Adrien from Ledger here, we don't yet support v2 but it is on the roadmap

**Comment by adrienlacombe** [comment]
Feel free to subscribe to https://github.com/LedgerHQ/wallet-connect-live-app/issues/8
